templates/jobs/configmap-utils.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{- include "graphdb.labels" . | nindent 4 }}
   annotations:
-    "helm.sh/hook": pre-install, pre-upgrade, pre-rollback, post-install, post-upgrade, post-rollback
+    "helm.sh/hook": post-install, post-upgrade, post-rollback
```

Comment:
I suppose this fixes `helm install --wait ....`?

Reply:
When ArgoCD syncs these pre hooks, it will wait for their completion, in particular the scale-down-job which is impossible to complete because the rest of the resources haven't been created yet.